agent/php_execute.c

```diff
@@ -1974,7 +1983,7 @@ static void nr_php_instrument_func_end(NR_EXECUTE_PROTO) {
    * been created if we are recording and if wraprec is set or if tt is greater
    * than 0.
```

Comment:
See [this comment](https://github.com/newrelic/newrelic-php-agent/pull/767/files#r1471534153).

Reply:
addressed in said comment in previous PR